dietpi.txt

```diff
@@ -260,6 +260,13 @@ SOFTWARE_HOMEASSISTANT_APT_DEPS=
 #	Add Python modules with version string at best, e.g.: firstModule==1.2.3 secondModule==4.5.6
 SOFTWARE_HOMEASSISTANT_PIP_DEPS=
 
+# K3s
+# Command with flags to use for launching K3s in the service
+# The value of this variable is copied directly into the INSTALL_K3S_EXEC environment variable before
+# running the k3s installer.
+# https://rancher.com/docs/k3s/latest/en/installation/install-options/#options-for-installation-with-script
+SOFTWARE_K3S_EXEC=
```

Comment:
Shall we add a note about the `/boot/dietpi-k3s.yaml` possibility here as well? When users edit `dietpi.txt` already, nice to know files files you can pre-create for installs 🙂.

Reply:
Yeah, good point. I'll add it.